### ISSUE TITLE
PLY imports are broken

### ISSUE THREAD

**Issue by dhritzkiv** [Matter-and-Form/assimp#3]
The server is currently complaining about malformed OBJs created by assimp from PLYs. I believe this is from the latest PLY code (f84851e8931fb04e7074d487c91c3b3faa566994) in assimp. Could it be the PLY code is broken, or the rebase went wrong?

Here is the PLY: [laurana500.ply.zip](https://github.com/Matter-and-Form/assimp/files/1105803/laurana500.ply.zip)

The contents of the OBJ look like this:

```obj
# File produced by Open Asset Import Library (http://www.assimp.sf.net)
# (assimp v3.3.674264542)

# 1 vertex positions
v  0 0 0

# 0 UV coordinates

# 0 vertex normals

# Mesh '' with 494 faces
p  1
p  1
p  1
p  1
p  1
p  1
p  1
p  1
p  1
p  1
#the rest is the same
```

The gltf created in the process is as such:

```json
{
    "asset": {
        "version": "1",
        "generator": "Open Asset Import Library (assimp v3.3.674264542)"
    },
    "accessors": {
        "mesh_accessor": {
            "bufferView": "mesh_view",
            "byteOffset": 0,
            "byteStride": 0,
            "componentType": 5126,
            "count": 1482,
            "type": "VEC3",
            "max": [
                0.0,
                0.0,
                0.0
            ],
            "min": [
                0.0,
                0.0,
                0.0
            ]
        },
        "mesh_accessor_0": {
            "bufferView": "mesh_view_0",
            "byteOffset": 0,
            "byteStride": 0,
            "componentType": 5123,
            "count": 494,
            "type": "SCALAR",
            "max": [
                493.0
            ],
            "min": [
                0.0
            ]
        }
    },
    "buffers": {
        "595275ce9ff8ead8c8a37f3a_595275cf7d6b25d8c94a96cf_100lod": {
            "byteLength": 18772,
            "type": "arraybuffer",
            "uri": "595275ce9ff8ead8c8a37f3a_595275d17d6b25d8c94a96d0_100lod.bin"
        }
    },
    "bufferViews": {
        "mesh_view": {
            "buffer": "595275ce9ff8ead8c8a37f3a_595275cf7d6b25d8c94a96cf_100lod",
            "byteOffset": 0,
            "byteLength": 17784,
            "target": 34962
        },
        "mesh_view_0": {
            "buffer": "595275ce9ff8ead8c8a37f3a_595275cf7d6b25d8c94a96cf_100lod",
            "byteOffset": 17784,
            "byteLength": 988,
            "target": 34963
        }
    },
    "meshes": {
        "mesh": {
            "name": "",
            "primitives": [
                {
                    "mode": 0,
                    "indices": "mesh_accessor_0",
                    "attributes": {
                        "POSITION": "mesh_accessor"
                    }
                }
            ]
        }
    },
    "nodes": {
        "node": {
            "meshes": [
                "mesh"
            ]
        }
    },
    "scenes": {
        "defaultScene": {
            "nodes": [
                "node"
            ]
        }
    },
    "scene": "defaultScene"
}
```

Using assimp built on the gltf-shader branch with is rebased on default-material, which is recently rebased on master

**Comment by johnmaf** [comment]
I can't reproduce this with the decimator pointing to https://github.com/johnmaf/assimp/commit/e722ea58d6304548dfae48e64123a9b76acd61bf and running `./bin/decimator laurana500.ply laurana.obj`

**Comment by dhritzkiv** [comment]
Issue opened in assimp/assimp#1324

**Comment by dhritzkiv** [comment]
Fixed in assimp/assimp#1534
